index.src.html

```diff
@@ -89,13 +89,13 @@
 
     <ul>
       <li>User goes to example.com on their laptop, is guided through a flow to create and register a credential on their phone.</li>
-      <li>User employs a FIDO 2.0 credential as described above to authorize a single transaction, such as a payment or other financial transaction.</li>
+      <li>User employs a scoped credential as described above to authorize a single transaction, such as a payment or other financial transaction.</li>
     </ul>
 
 
 # Conformance # {#conformance}
 
-  <p>This specification defines criteria for a <dfn>conforming user agent</dfn>. A user agent MUST behave as described in this specification in order to be considered conformant. User agents MAY implement algorithms given in this specification in any way desired, so long as the end result is indistinguishable from the result that would be obtained by the specification's algorithms. A conforming FIDO Credential API user agent MUST also be a conforming implementation of the IDL fragments of this specification, as described in the “Web IDL” specification. [[!WebIDL-1]]</p>
+  <p>This specification defines criteria for a <dfn>conforming user agent</dfn>. A user agent MUST behave as described in this specification in order to be considered conformant. User agents MAY implement algorithms given in this specification in any way desired, so long as the end result is indistinguishable from the result that would be obtained by the specification's algorithms. A conforming Scoped Credential API user agent MUST also be a conforming implementation of the IDL fragments of this specification, as described in the “Web IDL” specification. [[!WebIDL-1]]</p>
```

Comment:
It seems like our terminology is that we have a Web Authentication API which uses Scoped Credentials. So when we refer to the API it should be as the "Web Authentication API" not as the "Scoped Credential API".